src/components/button/index.jsx

```diff
@@ -48,7 +49,7 @@ class Button extends React.PureComponent {
         const ButtonWrapper = ({children, ...rest}) => {
             return href
                 ? <a {...rest} href={href}>{children}</a>
-                : <button {...rest} onClick={onClick} type={type}>{children}</button>
+                : <button {...rest} aria-label={ariaLabel} onClick={onClick} type={type}>{children}</button>
```

Comment:
Lets put this on the ButtonWrapper in the `render` method

Reply:
instead of the `render` method I think you meant to say the return method for the Button? Technically it is in the render method.

Reply:
yes, that's what I meant haha